test/transforms.jl

```diff
@@ -685,6 +685,22 @@
     @test t == tₒ
   end
 
+  @testset "Coerce" begin
+    x1 = [1.0, 2.0, 3.0, 4.0, 5.0]
+    x2 = [1.0, 2.0, 3.0, 4.0, 5.0]
+    x3 = [5.0, 5.0, 5.0, 5.0, 5.0]
+    t = Table(;x1, x2, x3)
+
+    # apply test
+    n, c = apply(Coerce(:x1=>Count, :x2=>Count), t)
```

Comment:
Please create the transform in a separate line as in other tests. Call it T for example.